src/edu/mills/cs180a/RomanNumeralTest.java

```diff
@@ -25,4 +25,10 @@ public void equals_False_X10() {
     Integer i = Integer.valueOf(10);
     assertNotEquals(rnX, i);
   }
+
+  @Test
+  public void reflective_true_XX_X10() {
+    RomanNumeral rn10 = new RomanNumeral(10);
+    assertEquals(rn10, rnX);
```

Comment:
This is a fine test, but reflexivity would imply: ```assertEquals(rnX, rnX);```.

Reply:
Oops. Any recommendations for naming? I'm drawing a blank.

Reply:
It's not transitive, is it?

Reply:
I'm not sure why there are so many Xs. Why not end it `X10` or `X_10`?

I'll admit I don't have the hang of this naming convention yet.